src/components/ChatApp/Settings/MobileTab.react.js

```diff
@@ -0,0 +1,164 @@
+import React, { Component } from 'react';
+import Translate from '../../Translate/Translate.react';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import TextField from 'material-ui/TextField';
+import PropTypes from 'prop-types';
+
+class MobileTab extends Component {
+  render() {
```

Comment:
Use functional component here, as we do not need to declare it as a React component